bcgov/moh-prime
# Feature/prime 1757 edit paper bday

Ability to edit a Unlinked Paper enrolments date of birth through an adjudicator action.

To test

- Create a paper enrolment
- Go to an admin screen with adjudicator actions
- Select Change Date of Birth
- Select a new date of Birth
- Save
- See updated Date of Birth in on overview page in card

- Link the paper enrolment to an enrollee
- Check that option no longer an action
